### DISCUSSION TITLE
How to delete part of a graphana dashboard.

### DISCUSSION THREAD

**Discussion by marcbaier** [teslamate-org/teslamate#3210] [Q&A] [answered]
Hi - i have the problem that after switching cars, i just cant git rid of the additional entry that is being shown in my Teslamate Grafana overview dashboard. Is it possible to overwrite the Grafana dashboard with a new version that does not show the info for my old "null" car ? Grafana allows me to save a separat JSON file which I could just use as a new version or not ?

Thanks for any help
Cheers
Marc

[](url)
<img width="1210" alt="Screen Shot 2023-05-22 at 10 56 56 AM" src="https://github.com/adriankumpf/teslamate/assets/97667985/56ac042b-f021-49ea-be9a-223da1698707">

**Comment by CarlosCuezva** [comment]
Currently not possible. You should delete the information of the old car from the database to avoid it showing up in Grafana.

**Reply by marcbaier** [reply]
thanks carlos - i have not yet figured out how to do that. (how to delete data that is in the docker container on my synology nas)

**Reply by CarlosCuezva** [reply]
In this topic you can obtain information about delete your old car

https://github.com/adriankumpf/teslamate/discussions/3074

Also, you can do it connecting to the database with a program, for example pgAdmin

**Reply by marcbaier** [reply]
thanks - maybe you can help. i tried to run the command "sudo docker-compose exec database psql teslamate teslamate" from the docker folder on my NAS. But it does not work. 
![Uploading Screen Shot 2023-05-23 at 09.25.55 AM.png…]()

**Reply by CarlosCuezva** [reply]
I can't view the image, can you upload again?

**Reply by marcbaier** [reply]
Am getting this error:         

Can't find a suitable configuration file in this directory or any parent. Are you in the right directory?

<img width="705" alt="Screen Shot 2023-05-23 at 09 25 55 AM" src="https://github.com/adriankumpf/teslamate/assets/97667985/d86936d5-18df-413a-9625-cd2df537cd84">

**Reply by marcbaier** [reply]
i also installed pgAdmin if that helps

**Reply by CrowMash** [reply]
You need to execute this command in the directory where you put the docker-compose.yml file.

**Reply by CarlosCuezva** [reply]
> i also installed pgAdmin if that helps

To be able to connect to the database with pgAdmin, you first have to edit the docker-compose.yml file and open the port as in the image

![image](https://github.com/adriankumpf/teslamate/assets/5815421/cbdb71fd-ee81-4da7-a71c-3bd0f28ea894)

**Reply by marcbaier** [reply]
ok great - now I am getting a prompt: teslamate=#

**Reply by marcbaier** [reply]
carlos - very stupid question: how do i edit the .yml file ? i think i now know that it is in the @docker folder.

**Reply by marcbaier** [reply]
ok i opened the file. do i understand correctly that I need to ADD the ports details ? my file looks like this: 

database:
    image: postgres:13
    restart: always
    environment:
      - POSTGRES_USER=xxx
      - POSTGRES_PASSWORD=xxx
      - POSTGRES_DB=xxx
    volumes:
      - teslamate-db:/var/lib/postgresql/data

  grafana:
    image: teslamate/grafana:latest
    restart: always
    environment:
      - DATABASE_USER=xxx
      - DATABASE_PASS=xxx
      - DATABASE_NAME=xxx
      - DATABASE_HOST=database
    ports:
      - 3000:3000
    volumes:
      - teslamate-grafana-data:/var/lib/grafana

  mosquitto:
    image: eclipse-mosquitto:2
    restart: always
    command: mosquitto -c /mosquitto-no-auth.conf
    # ports:
    #   - 1883:1883
    volumes:
      - mosquitto-conf:/mosquitto/config
      - mosquitto-data:/mosquitto/data

volumes:
  teslamate-db:
  teslamate-grafana-data:
  mosquitto-conf:
  mosquitto-data:

**Reply by marcbaier** [reply, marked as answer]
SOLVED. I was able to do it manually with the instructions once I figured out how to show and select the old cars ID. Thanks a lot for your help guys.
